cookbook/core/Dockerfile

```diff
@@ -33,3 +33,5 @@ COPY core /root/core
 # when registering tasks, workflows, and launch plans
 ARG tag
 ENV FLYTE_INTERNAL_IMAGE $tag
+
+RUN pip install -U https://github.com/flyteorg/flytekit/archive/41c702dca988b3410f7d1b8cc3ffe1ab8d5d2879.zip#egg=flytekit
```

Comment:
reminder to get rid of this...

Reply:
yup, we have to go through a bunch of releases to do this